MDANSE/Src/MDANSE/Framework/Jobs/VanHoveFunctionSelf.py

```diff
@@ -119,7 +119,7 @@ class VanHoveFunctionSelf(IJob):
     )
     settings["output_files"] = (
         "OutputFilesConfigurator",
-        {"formats": ["MDAFormat", "TextFormat"]},
+        {"formats": ["MDAFormat", "TextFormat", "FileInMemory"]},
```

Comment:
Do these just want to be defined as default in `OutputFilesConfigurator` and only overridden if different?